core/include/webview/detail/backends/cocoa_webkit.hh

```diff
@@ -489,7 +460,7 @@ private:
       objc::msg_send<void>(app, "activateIgnoringOtherApps:"_sel, YES);
     }
 
-    set_up_window();
+    window_init();
```

Comment:
In the constructor, I can easily understand that `window_init(window)` means that we're going to perform window setup/initialization, potentially with an existing window.

What does it mean when `window_init()` is called again from `on_application_did_finish_launching()` without a parameter (`= nullptr`)?

Reply:
It means that by the time `on_application_did_finish_launching` is called, ` m_window = static_cast<id>(window)` was already set, so the `window` parameter is moot (ie. there is no further reference to `window` in `window_init`). Passing `nullptr` is thus rational.

If the the user DID pass a `window` into the class constructor -> `window_init(window)`, `!m_window` will be false on the second call to `window_init`, so m_window is guarded from being reset.